packages/iris-grid/src/IrisGridTreeTableModel.ts

```diff
@@ -181,9 +181,8 @@ class IrisGridTreeTableModel extends IrisGridTableModelTemplate<
           c <= intersection.endColumn;
           c += 1
         ) {
-          assertNotNull(formatValue);
           resultRow.push(
-            formatValue(viewportRow.data.get(c)?.value, this.columns[c])
+            formatValue?.(viewportRow.data.get(c)?.value, this.columns[c])
```

Comment:
Seems if `formatValue` is not defined, there is no point in entering the for loop at all. I would check before the loop.

Reply:
Good catch, just tested and it seems like this also works, but definitely more efficient (since we are not going into the loop if its not defined anymore). Pushed change